client/App/Stores/Gps/Actions.js

```diff
@@ -0,0 +1,15 @@
+import { createActions } from 'reduxsauce'
+
+const { Types, Creators } = createActions({
+  // Fetch user informations
```

Comment:
change comment to be gps instead of user

Reply:
done! :)